racket/src/cfg-cs

```diff
@@ -2272,6 +2272,9 @@ if test "${enable_racket}" = "auto" ; then
 fi
 RUN_LOCAL_RACKET="local/cs/c/racketcs"
 CONFIGURE_LOCAL_RACKET=--enable-cs
+if test "${enable_pb}" = "yes" ; then
+  CONFIGURE_LOCAL_RACKET="${CONFIGURE_LOCAL_RACKET} --enable-pb"
```

Comment:
I see why this makes the build work, but I think the right repair involves corrections to the main CS `configure.ac` and `Makefile.in`. (The general constraint on `--enable-racket=auto` mode is that the host Racket build must work without any configuration, so that means the host build shouldn't require `--enable-pb` itself, so that's why this workaround isn't really the right approach.)